tools/run_tests/xds_k8s_test_driver/config/grpc-testing.cfg

```diff
@@ -5,3 +5,5 @@
 --private_api_key_secret_name=projects/830293263384/secrets/xds-interop-tests-private-api-access-key
 # Randomize xds port.
 --server_xds_port=0
+# ResultStore UI doesn't support 256 colors.
+--color_style=ansi16
```

Comment:
Can we document the supported options somewhere?

Reply:
`--help` shows all of them:

>   --color_style: <ansi16|default|emacs|friendly|colorful|autumn|murphy|manni|material|monokai|perldoc|pastie|borland|trac|native|fruity|bw|vim|vs|tango|rrt|xcode|igor|paraiso-light|paraiso-dark|lovelace|algol|algol_nu|arduino|rainbow_dash|abap|solarized-dark|solarized-light|sas|stata|stata-light|stata-dark|inkpot|zenburn|gruvbox-dark|gruvbox-light>: Color
    styles for terminals supporting 256 colors. Use ansi16 style for terminals supporting 8/16 colors